### PR TITLE
pocketbook: support "extended" key events

### PR DESCRIPTION
**Pull request by benoit-pierre**
Only for device running with SDK >= 6.

This cleanup and update the inkview cdecls, dropping unused declarations and adding declarations for "extended" key events (`EVT_KEYPRESS_EXT` and the like).

For now, "standard" keycodes are negated to avoid conflicts with "extended" (regular linux input) keycodes.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader-base/1971)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]

File: ffi/input_pocketbook.lua [hunk 1]
```diff
@@ -57,90 +60,36 @@ local pb_event_map = {
...
-    [C.EVT_SCANSTOPPED] = "EVT_SCANSTOPPED",
-    [C.EVT_POSTPONE_TIMED_POWEROFF] = "EVT_POSTPONE_TIMED_POWEROFF",
-    [C.EVT_FRAME_ACTIVATED] = "EVT_FRAME_ACTIVATED",
-    [C.EVT_FRAME_DEACTIVATED] = "EVT_FRAME_DEACTIVATED",
-    [C.EVT_READ_PROGRESS_CHANGED] = "EVT_READ_PROGRESS_CHANGED",
-    [C.EVT_DUMP_BITMAPS_DEBUG_INFO] = "EVT_DUMP_BITMAPS_DEBUG_INFO",
-    [C.EVT_NET_CONNECTED] = "EVT_NET_CONNECTED",
-    [C.EVT_NET_DISCONNECTED] = "EVT_NET_DISCONNECTED",
-    [C.EVT_NET_FOUND_NEW_FW] = "EVT_NET_FOUND_NEW_FW",
-    [C.EVT_SYNTH_POSITION] = "EVT_SYNTH_POSITION",
```
Comment (NiLuJe) [inline ffi/input_pocketbook.lua:141]: I am sort of attached to the full translation map, it makes reading PB logs way more useful ;).

Comment (benoit-pierre) [reply ffi/input_pocketbook.lua]: Amended.

**Review by NiLuJe — APPROVED** [review]
> Reviewed 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/koreader/koreader-base/1971)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @benoit-pierre)

**Review by NiLuJe — APPROVED** [review]
> Reviewed 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/koreader/koreader-base/1971)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @benoit-pierre)

**Pull request merged**